package.json

```diff
@@ -54,7 +54,6 @@
     "ember-collection": "1.0.0-alpha.5",
     "ember-data": "2.7.0",
     "ember-data-factory-guy": "^2.6.1",
-    "ember-disable-prototype-extensions": "^1.0.0",
```

Comment:
Thanks.

I recently had addon unit tests fail completely because the addon code was using prototype extensions. (and ember unit tests turn those extensions off) 

So as I added tests, more stuff was breaking- it was certainly no fun to track down, and I'd love to save someone else that trouble. Appreciate your help and improvements to the code.